src/other/Table.js

```diff
@@ -27,6 +27,7 @@
     Table.prototype.publishProxy("itemsPerPage", "_paginator");
     Table.prototype.publishProxy("pageNumber", "_paginator", "pageNumber",1);
     Table.prototype.publishProxy("adjacentPages", "_paginator");
+    Table.prototype.publish("numRows", null, "number", "Total number or rows of data to be displayed in the table",null,{tags:["Private"]});
```

Comment:
minor:  Can you rename this to topN (as this is a common term for this feature in ECL).